tools/bin/hawq

```diff
@@ -143,7 +143,7 @@ def main():
         cmd = "%s; hawq_ctl %s %s" % (source_hawq_env, hawq_command, sub_args)
         result = local_run(cmd)
     elif hawq_command == "config":
-        if second_arg in ['', 'help', '--help']:
+        if second_arg in ['', 'help', '--help', '-h']:
```

Comment:
Also fix for "hawq activate"?

   elif hawq_command == "activate":
        if second_arg in ['', 'help', '--help']:
            print ACTIVE_HELP
            sys.exit(1)
        cmd = "%s; hawq_ctl %s %s" % (source_hawq_env, hawq_command, sub_args)
        result = local_run(cmd)